certificate-policy.md

```diff
@@ -924,40 +924,143 @@ Based on the Risk Assessment, the CA SHALL develop, implement, and maintain a se
+
+RA equipment SHALL be protected from unauthorized access while the cryptographic module is installed and activated.  The RA SHALL implement physical access controls to reduce the risk of equipment tampering even when the cryptographic module is not installed and activated.  These security mechanisms SHALL be commensurate with the level of threat in the RA equipment environment.
 
 ### 5.1.3 Power and air conditioning
+The CA SHALL have backup capability sufficient to lock out input, finish any pending actions, and record the state of the equipment automatically before lack of power or air conditioning causes a shutdown.  
+
+The repositories (containing CA certificates and CRLs) SHALL be provided with uninterrupted power sufficient for a minimum of 6 hours operation in the absence of commercial power, to maintain availability and avoid denial of service.
 
 ### 5.1.4 Water exposures
+CA equipment SHALL be installed such that it is not in danger of exposure to water (e.g., on tables or elevated floors).
+Potential water damage from fire prevention and protection measures (e.g., sprinkler systems) are excluded from this requirement.
 
 ### 5.1.5 Fire prevention and protection
+No Stipulation
 
 ### 5.1.6 Media storage
+Media SHALL be stored so as to protect them from accidental damage (e.g., water, fire, or electromagnetic) and unauthorized physical access.
 
 ### 5.1.7 Waste disposal
+Sensitive media and documentation that are no longer needed for operations SHALL be destroyed in a secure manner.  For example, sensitive paper documentation SHALL be shredded, burned, or otherwise rendered unrecoverable.
 
 ### 5.1.8 Off-site backup
+Full system backups sufficient to recover from system failure SHALL be made on a periodic schedule.  Backups are to be performed and stored off-site not less than once per week.  At least one full backup copy SHALL be stored at an off-site location (separate from CA equipment).  Only the latest full backup need be retained.  The backup SHALL be stored at a site with physical and procedural controls commensurate to that of the operational CA.
 
 ## 5.2 Procedural controls
 
 ### 5.2.1 Trusted roles
+A trusted role is one whose incumbent performs functions that can introduce security problems if not carried out properly, whether accidentally or maliciously.  
+The requirements of this policy are defined in terms of four roles.   
+
+1.	Administrator – authorized to install, configure, and maintain the CA; establish and maintain system accounts; configure audit parameters; and generate component keys. 
+2.	Officer – authorized to request or approve certificate issuance and revocations. 
```

Comment:
I think the CA Officer needs to be clarified here and in the other sections?  This does not need to be a human (for one) and can be automated.